### PR TITLE
test: Skip flakey AWS plugin test

### PR DESCRIPTION
**Pull request by mattgauntseo-sentry**
This test is causing a lot of events

- https://sentry.sentry.io/issues/3892447489/?project=2423079
- https://sentry.sentry.io/issues/3901571639/?project=2423079

GitHub Issue: https://github.com/getsentry/sentry/issues/44858

### CODE REVIEW

File: tests/sentry_plugins/amazon_sqs/test_plugin.py [hunk 1]
```diff
@@ -134,6 +134,9 @@ def test_use_s3_bucket(self, mock_client):
 
     @patch("sentry_plugins.amazon_sqs.plugin.logger")
     @patch("boto3.client")
+    @pytest.mark.skip(
+        reason="Flakey: https://sentry.sentry.io/issues/3901571639/?project=2423079 and https://sentry.sentry.io/issues/3901571639/?project=2423079"
+    )
```
Comment (asottile-sentry) [inline tests/sentry_plugins/amazon_sqs/test_plugin.py:139]: let's make sure to create a ticket so it gets fixed at some point -- I want to say ecosystem owns this?

Comment (mattgauntseo-sentry) [reply tests/sentry_plugins/amazon_sqs/test_plugin.py:139]: Done - added issue to description. I tried to find an owner and I came up blank.

Comment (asottile-sentry) [reply tests/sentry_plugins/amazon_sqs/test_plugin.py:139]: ideally we'd have the issue number in the skipped message -- ala https://github.com/getsentry/rfcs/blob/main/text/0027-manual-disabling-of-flaky-tests.md

**Review by asottile-sentry — APPROVED** [review]

**Pull request merged**